.github/workflows/ci-build-image.yml

```diff
@@ -18,10 +18,6 @@ on:
       image-version:
         description: the Cryostat application version that will be built
         value: ${{ jobs.get-pom-properties.outputs.image-version }}
-    secrets:
-      GH_PKGS_READ_TOKEN:
-        required: true
-        description: read-only token for pulling artifacts from GitHub Packages
```

Comment:
I am reading from this:

https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry

Looks like `GITHUB_TOKEN` should work as long as they have `packages:read` scope on it, which it should since its a `issue_comment` event.

Reply:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token